chart/values.yaml

```diff
@@ -0,0 +1,235 @@
+# Default values for policy-bot.
+# This is a YAML-formatted file.
+# See README.md for detailed documentation.
+
+## @section Deployment Settings
+
+## @param replicaCount Number of replicas to deploy. policy-bot is stateless and safe to run multiple instances.
+replicaCount: 2
+
+## @param image.repository Docker image repository
+## @param image.pullPolicy Image pull policy (Always, IfNotPresent, Never)
+## @param image.tag Overrides the image tag whose default is the chart appVersion
+image:
+  repository: palantirtechnologies/policy-bot
+  pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
We should set a real tag here that is pinned to the release that is about to be cut. We either need to see if godel can do this without failing a verify or update how we do releases.